src/main/resources/application.properties

```diff
@@ -1,13 +0,0 @@
-#Server
-server.port=8080
```

Comment:
I dont see your `application.properties` or `application.yaml` file for Kafka configurations.

Reply:
Hey sorry for the inconvenience. Actually I forgot to remove the application.properties from .gitignore file. Please Review it again. I have added the file in the latest commit.